src/collections/main/Main.java

```diff
@@ -0,0 +1,56 @@
+package collections.main;
+
+import collections.comparator.StudentNameAscendingAndAgeDescendingComparator;
+import collections.model.*;
+import collections.model.Student;
+
+import java.util.*;
+
+import static collections.model.StoredStudents.*;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Student ivan = new Student.Builder("Ivan")
+                .setLastName("Gyulumyan")
+                .setAge(20)
+                .build();
+        Student anna = new Student("Anna", "Martirosyan", 18);
+        Student hakob = new Student("Hakob", "Petrosyan", 23);
+        Student arthur = new Student("Arthur", "Grigoryan", 22);
+        Student narek = new Student("Narek", "Asatryan", 21);
+        Student narek22 = new Student("Narek", "Asatryan", 22);
+        Student narek20 = new Student("Narek", "Asatryan", 20);
+        Student ivan18 = new Student("Ivan", "Gyulumyan", 18);
+
+        StoredStudents temp = new StoredStudents();
+        temp.initDynamicArray(ivan, anna, narek, arthur);
+        temp.initLinkedList(anna, narek, ivan, hakob);
+
+        doublyLinkedList.push(narek);
+        doublyLinkedList.addLast(ivan);
+        doublyLinkedList.print();
+        doublyLinkedList.pop();
+        doublyLinkedList.print();
+        doublyLinkedList.removeLast();
+        doublyLinkedList.print();
+
+        ArrayList<Student> arrayList = new ArrayList<>();
+        arrayList.add(anna);
+        arrayList.add(ivan);
+        arrayList.add(hakob);
+        arrayList.add(narek20);
+        arrayList.add(narek);
+        arrayList.add(arthur);
+        arrayList.add(narek22);
+        arrayList.add(ivan18);
+        StudentNameAscendingAndAgeDescendingComparator cmp = new StudentNameAscendingAndAgeDescendingComparator();
+        arrayList.sort(cmp);
+        for (Student tmp : arrayList) {
+            System.out.println(tmp);
+        }
+
+        dynamicArrayOfStudents.get(-11);
```

Comment:
I tried to remove `anna = new Student("Anna", "Martirosyan", 18)` from `dynamicArrayOfStudents` and it returned false, why?
`dynamicArrayOfStudents.remove(new Student("Anna", "Martirosyan", 18))`

Reply:
it returns false only when the object we want to remove doesn't exists in dynamic array, in this case we want to remove new object that doesn't exists in our dynamic array, so it returns false

Reply:
What do you mean by `new object`? Aren't these two objects the same? 
added: `new Student("Anna", "Martirosyan", 18)`
removed: `new Student("Anna", "Martirosyan", 18)`

Reply:
this objects have different references, so they are different persons, I haven't overrided equals method for Students.

Reply:
Right, and that is the exact point I am trying to make clear for you. Unless you properly override the equals method, your remove method will not work properly, e.g. suppose you created a `dynamicArrayOfStudents` somewhere in your project, and tried to use it in any other place, you have to send them not only your `dynamicArrayOfStudents` but also all references you created, in order your remove method work properly.